### DISCUSSION TITLE
2FA adding

### DISCUSSION THREAD

**Discussion by pratovv** [verdaccio/verdaccio#3547] [Q&A] [answered]
I'm working to add 2FA to verdaccio. Could you point to the places in code where adjustments are required in order to add support for this feature?

**Comment by juanpicado** [comment, marked as answer]
Here is some context https://github.com/verdaccio/verdaccio/discussions/913

But I think first requires some research, verdaccio is an express.js app so I think any guide for it would help.

I haven't get into that topic so I have no suggestions at this point to provide.

**Reply by pratovv** [reply]
Ok,I will do research.Thank you for your answer.

**Reply by juanpicado** [reply]
Feel free to post here or via discord chat there is a channel for contributions.
